ichnaea/api/locate/tests/base.py

```diff
@@ -411,6 +454,7 @@ def test_gzip(self, app, data_queues):
         headers = {"Content-Encoding": "gzip"}
         res = self._call(app, body=body, headers=headers, method="post", status=404)
         self.check_response(data_queues, res, "not_found")
+        assert logs.entry["wifi"] == logs.entry["wifi_valid"] == 2
```

Comment:
How does pytest handle this assertion failing? Does it show helpful output?

Reply:
Here's some cases. For the assertion:

```
assert logs.entry["wifi"] == logs.entry["wifi_valid"] == 2
```

``wifi=1``, ``wifi_valid=2``:

```
Traceback (most recent call last):
  File "/app/ichnaea/api/locate/tests/base.py", line 458, in test_gzip
    assert logs.entry["wifi"] == logs.entry["wifi_valid"] == 2
AssertionError: assert 1 == 2
```

``wifi=2``, ``wifi_valid=1``:

```
Traceback (most recent call last):
  File "/app/ichnaea/api/locate/tests/base.py", line 458, in test_gzip
    assert logs.entry["wifi"] == logs.entry["wifi_valid"] == 2
AssertionError: assert 2 == 1
```

Not too clear. I'm going to replace these with just testing ``wifi_valid``:

```
assert logs.entry["wifi_valid"] == 2
```

If ``wifi_valid=1``:

```
Traceback (most recent call last):
  File "/app/ichnaea/api/locate/tests/base.py", line 458, in test_gzip
    assert logs.entry["wifi_valid"] == 2
AssertionError: assert 1 == 2
```

Reply:
That's good to know! Thank you for testing!